.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should not be submitted here on GitHub, it should be private on you machine.